database/connection.py

```diff
@@ -1,15 +1,23 @@
 import pymysql
+import pymysql.cursors
+import cx_Oracle
 
 #database connection
-connection = pymysql.connect(host="localhost", user="root", passwd="", database="test")
-cursor = connection.cursor()
-select_tbl= "select * from tbl_employee "
-cursor.execute(select_tbl)
+def db_connect():
+
```

Comment:
Remove extra spaces.

Reply:
This code already removed from file. please see the updated file